package.json

```diff
@@ -22,12 +24,15 @@
     "url": "git@github.com:bjrmatos/jsreport-jade.git"
   },
   "dependencies": {
+    "in-publish": "2.0.0",
```

Comment:
I'm totally confused by this. :)

Apparently `npm install` runs `prepublish` script (https://github.com/npm/npm/issues/3059) 
That is why we use https://github.com/iarna/in-publish to skip the prepublish script during `npm install`. (I took this from your repositories, thanks :) )
If I run `npm install`, I don't have `devDependencies` installed, for sure not for the nested dependencies, so I guessed I need to put `in-publish` to `dependencies`. As far I remember I even got error in travis: `in-publish not found` before I moved it to `devDepndencies`.

I'm probably wrong, because I see you have `in-publish` in `devDependencies`  https://github.com/bjrmatos/electron-workers/blob/master/package.json#L43 and it works nice. Do you please  have an explanation where my thoughts go wrong?